### PR TITLE
Update README to show use of triggerPromise()

### LINKED ISSUE #259: someAction.then().catch() not working
the [README](https://github.com/spoike/refluxjs#asynchronous-actions-as-promises) mentions that async actions can be used as Promises with `then/catch()` methods, but that doesn't seem to work. when I try to call the `then()` method of one of my actions my browser complains: `Uncaught TypeError: Cannot read property 'then' of undefined`.

### PR DESCRIPTION
**Pull request by patrick-webs**
This fixes #259 which is a result of the README documenting this feature incorrectly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -219,7 +219,7 @@ var RequestStore = Reflux.createStore({
 Then, on the server, you could use promises to make the request and either render or serve an error:
 
 ```javascript
-makeRequest('/api/something').then(function(body) {
+makeRequest('/api/something').triggerPromise().then(function(body) {
```
Comment (jankuca) [inline README.md]: This should say `makeRequest.triggerPromise('/api/something').then(....`

Comment (patrick-webs) [inline README.md]: fixed, thanks

**Comment by ivan-kleshnin** [comment]
Is this line "on server" correct?

**Comment by cmwelsh** [comment]
:+1: for implementing rather than requesting (like I did)

**Comment by cmwelsh** [comment]
However, I am in favor of ditching this `triggerPromise` syntax and adding an auto-promise option for actions (some people expect this to already happen). For example, if `Reflux.createAction({ sync: true})` returns a synchronous action, perhaps `Reflux.createAction({ promise: true})` could create a promise action.

**Comment by spoike** [comment]
Well for now, it is easier to fix the documentation.

I agree though with @cmwelsh, `triggerPromise` has tripped some users up.

**Pull request merged**